mantidproject/mantid
# Compress weighted events

This is the long awaited "fat events" implementation. In reality it adds two extra parameters to `CompressEvents` to allow for keeping the pulse time.

**To test:**

Load an event workspace and see what happens when you compress it with pulse-times. Various tests have been added, but there more testing won't hurt it.

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
